Resources/Prototypes/Entities/Mobs/Player/arachne.yml

```diff
@@ -5,6 +5,11 @@
   id: MobArachne
   components:
   - type: CombatMode
+  - type: InteractionPopup
```

Comment:
why does this re-add hugging? probably explained in the description i didn't read

Reply:
Oh fuck did I forget to remove those...